### ISSUE TITLE
Delta Audio Blast

### ISSUE THREAD

**Issue by P1neAppl** [rileytestut/Delta#236]
I'm on an iPhone 11 with Delta 1.4 installed.
This problem seems to happen with "respect silent mode" either on or off.
Whilst I'm playing a Delta game for an extended period of time, the audio of the game will suddenly blast/increase in volume in the middle of gameplay.
I wish I could give video proof, but it's so random I can't predict when it happens.

**Comment by shanegillio** [comment]
Hmm, are you getting this with all games?

**Comment by P1neAppl** [comment]
Yes, I am getting this in all games, for supposedly all systems too.

**Comment by shanegillio** [comment]
Can you tell me if you were using headphones or noticed anything else? The volume just increased without any screen indicator like the iPhone volume slider?

**Comment by P1neAppl** [comment]
> Can you tell me if you were using headphones or noticed anything else? The volume just increased without any screen indicator like the iPhone volume slider?

This has happened as I was using headphones, and it just blasts no matter what without a volume slider display. I keep "Respect Silent Mode" on for whenever it happens.

**Comment by P1neAppl** [comment]
> ok, I'll do some testing on 1.4 and check it out

Thanks and good luck
